projects/pipelines/testing/test.py

```diff
@@ -65,13 +65,17 @@ def wrapper(*args, **kwargs):
     return decorator
 # ---
 
-def customize_rhods():
-    if not config.ci_artifacts.get_config("rhods.operator.stop"):
-        return
-
-    run.run("oc scale deploy/rhods-operator --replicas=0 -n redhat-ods-operator")
-    time.sleep(10)
-
+def rhoai_down():
+    # Pause RHOAI by scaling replicas -> 0
+    if run.run(f'oc get deploy/rhods-operator -n redhat-ods-operator 2>/dev/null', check=False).returncode == 0:
+        run.run("oc patch deployment.apps/rhods-operator -n redhat-ods-operator --type merge -p '{\"spec\": {\"replicas\": 0}}'")
+        run.run("oc wait --for jsonpath='{.spec.replicas}'=0 deployment.apps/rhods-operator -n redhat-ods-operator --timeout=5m")
+
+def rhoai_up():
+    # Resume RHOAI by scaling replicas -> 1
+    if run.run(f'oc get deploy/rhods-operator -n redhat-ods-operator 2>/dev/null', check=False).returncode == 0:
+        run.run("oc patch deployment.apps/rhods-operator -n redhat-ods-operator --type merge -p '{\"spec\": {\"replicas\": 1}}'")
+        run.run("oc wait --for jsonpath='{.status.availableReplicas}'=1 deployment.apps/rhods-operator -n redhat-ods-operator --timeout=5m")
```

Comment:
this could have been stored in `projects/rhods/library/prepare_rhoai.py`. I'll move it next time I need it :)